src/model/entities/test/test-entity-factory.ts

```diff
@@ -0,0 +1,52 @@
+import { Rundown, RundownInterface } from '../rundown'
+import { Segment, SegmentInterface } from '../segment'
+import { Part, PartInterface } from '../part'
+import { Piece, PieceInterface } from '../piece'
+
+export class TestEntityFactory {
+  public static createRundown(rundownInterface?: Partial<RundownInterface>): Rundown {
+    if (!rundownInterface) {
+      rundownInterface = {} as RundownInterface
+    }
+
+    return new Rundown({
+      id: rundownInterface.id ?? 'rundownId' + Math.floor(Math.random()*1000),
+      name: rundownInterface.name ?? 'rundownName',
+      segments: rundownInterface.segments ?? []
+    } as RundownInterface)
```

Comment:
Isn't the purpose of this factory to fill in default values for non-provided values for the `XInterface` objects?

Reply:
Yes, but i chose to only add for the properties i make use at the moment, planning for people to add default values as they need them. Don't know if that is the correct approch to take, but at the same time having a value for all properties doesn't matter, if no test makes use of it.

Reply:
I do get that, but the effort needed to put in some default values is quite small.
Also if you use the spread operator to overwrite the default values, you will get more readable methods:
```typescript
  public static createRundown(rundownInterface: Partial<RundownInterface> = {}): Rundown {
    return new Rundown({
      id: 'rundownId' + Math.floor(Math.random()*1000),
      name: 'rundownName',
      segments: [],
      ...rundownInterface,
    } as RundownInterface)
  }

Reply:
Have changed to use the suggested method signature, and also added default values for a few more properties on each entity. Still not for all properties, as i frankly don't know what would be an ok default for a great deal of the newer properties introduced by Lindved.

Ps. Hope its okay that i had a small amount of fun with some of the default values :P
It is only used for tests anyways. If not, then i will begrudgingly change them.